ocs_ci/helpers/dr_helpers.py

```
@@ -2865,20 +2876,25 @@ def create_service_exporter(annotate=True):
     for cluster in managed_clusters:
         index = cluster.MULTICLUSTER["multicluster_index"]
         config.switch_ctx(index)
-        if (
+
+        if not (
             cluster.ENV_DATA.get("cluster_type").lower() == constants.HCI_CLIENT
-            or get_provider_service_type() == "NodePort"
+            and get_provider_service_type() == "NodePort"
```

Ruff (0.15.12)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)